src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManagerSpec.groovy

```diff
@@ -219,12 +244,14 @@ class LSFJobManagerSpec extends Specification {
         jobInfo.jobName == "ls -l"
         jobInfo.tool == new File("ls -l")
         jobInfo.jobID == new BEJobID("22005")
-        jobInfo.submitTime == ZonedDateTime.of(2017, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
+
+        // The year-parsing/inferrence is checked in another test. Here just take the parsed value.
+        jobInfo.submitTime == ZonedDateTime.of(jobInfo.submitTime.year, 12, 28, 19, 56, 0, 0, ZoneId.systemDefault())
```

Comment:
You are refering to jobInfo.submitTime in the .of method but you are assigning the jobInfo.submitTime field here. Does it really work?

Reply:
I don't understand what you mean. The test of the year field is in the new test method above. The test here takes a short-cut and just copies the actual year, because otherwise I would have to reimplement all the logic that I am testing in the new test method. And, yes, it works.